homeassistant/components/fronius/coordinator.py

```diff
@@ -155,14 +155,17 @@ async def _update_method(self) -> dict[SolarNetId, Any]:
                 data = await self.solar_net.fronius.current_inverter_data(
                     self.inverter_info.solar_net_id
                 )
+                data_3p = await self.solar_net.fronius.current_inverter_3p_data(
+                    self.inverter_info.solar_net_id
+                )
```

Comment:
For single phase inverters (or other that may not support this endpoint), is there a common error code returned - something different as when it is not producing? We could check this on first attempt and store a flag for it so we can skip the query in subsequent updates.

I think it would also be a good idea to have separated try-except blocks for the two requests.

Reply:
Since some Fronius devices are particularly sensible to HTTP queries, I think it would be wise to limit this if not needed. 
If you have a 3 phase inverter at hand, it would be easy to find out how this behaves eg. when not producing. For the turned off case, the first request for CommonInverterData would have failed already. If the first request was successful, I guess the 3PInverterData request can be done too - if it results in an empty body, maybe don't send it again next time.

Another way to limit, I could imagine, would be to check if any of the 6 new entities are active. If all are deactivated anyway, we can omit the request - the data wouldn't be used anyway.